book/website/community-handbook/newsletters.md

```diff
@@ -0,0 +1,26 @@
+# _The Turing Way_ Monthly Newsletters
+
+_The Turing Way_ team releases newsletters every month to share monthly updates with the community and beyond.
```

Comment:
Minor wording point, but there are two version of month in the short sentence, so maybe "to share latest updates"?